Makefile

```diff
@@ -161,6 +161,8 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+gen.codegen: codegen
+
```

Comment:
Is there a reason this one is separated by an empty line but others arent?

Reply:
@BlaineEXE wrote:
> Is there a reason this one is separated by an empty line but others arent?


No particular reason.

Will make this systematic as well.

Reply:
done.